lib/decorateWithMetrics.js

```diff
@@ -115,30 +127,55 @@ function toHumanReadableDuration(millis) {
  * Print extensive information on the dev console
  * max, min, average, total time for each method
  * and a table of individual calls
+ *
+ * @param {boolean} [raw=false] setting this to true will print raw instead of human friendly times
+ * Useful when you copy the printed table to excel and let excel do the number formatting
```

Comment:
Having the raw number in excel means you can also do sorting or other number related operations which aren't easily performed with a value like `"565.344ms"`